### ISSUE TITLE
Error executing the script

### ISSUE THREAD

**Issue by the-universal-linux-society** [Whomrx666/Xtracking#1]
When running the script i get the following error message : ValueError: bad marshal data (unknown type code)
Python seems to be having an issue understanding the code,or how to execute it.

**Comment by Whomrx666** [comment]
> When running the script i get the following error message : ValueError: bad marshal data (unknown type code)
> Python seems to be having an issue understanding the code,or how to execute it.

Yes, because I have locked it with Marshall so in some terminals it doesn't support Marshall, try other terminals like Termux or Kali Linux

**Comment by Whomrx666** [comment]
> When running the script i get the following error message : ValueError: bad marshal data (unknown type code)
> Python seems to be having an issue understanding the code,or how to execute it.

Yes, because I have locked it with Marshall so in some terminals it doesn't support Marshall, try other terminals like Termux or Kali Linux
